src/gui/org/deidentifier/arx/gui/model/ModelEvent.java

```diff
@@ -122,7 +122,10 @@
         COST_BENEFIT_MODEL,
         
         /** SELECTED_CLASS_VALUE */
-        SELECTED_CLASS_VALUE
+        SELECTED_CLASS_VALUE,
+        
+        /** STATISTICAL_CLASSIFIER */
+        STATISTICAL_CLASSIFIER
```

Comment:
It is likely that SELECTED_FEATURES_OR_CLASSES can be renamed and reused for this purpose, so that no additional event type is needed. Please check carefully.